easybuild/easyblocks/g/gromacs.py

```diff
@@ -324,34 +378,61 @@ def configure_step(self):
                     if not regex.search(out):
                         raise EasyBuildError("Pattern '%s' not found in GROMACS configuration output.", pattern)
 
+    def build_step(self):
+        """
+        Custom build step for GROMACS; Skip if CUDA is enabled and the current
+        iteration is for double precision
+        """
+
+        if self.is_double_precision_cuda_build():
+            self.log.info("skipping build step")
+            return
+
+        super(EB_GROMACS, self).build_step()
```

Comment:
Not a big fan of inline `return` statements, can be easily avoided here:

```python
if self.is_double_precision_cuda_build():
    self.log.info("skipping build step")
else:
    super(EB_GROMACS, self).build_step()

Reply:
Done